Tests/scripts/validate_files_structure.py

```diff
@@ -234,6 +264,9 @@ def validate_committed_files(branch_name):
         if re.match(PLAYBOOK_REGEX, file_path, re.IGNORECASE) or re.match(SCRIPT_REGEX, file_path, re.IGNORECASE) or re.match(TEST_PLAYBOOK_REGEX, file_path, re.IGNORECASE):
```

Comment:
since in the script the id is not at the root of the json, this should be with the integration and not the playbooks.
but in my opinion this should be the same function anyway (changed_id)